deploy/helm/sumologic/conf/logs/logs.conf

```diff
@@ -5,4 +5,5 @@
 </source>
 @include logs.source.containers.conf
 @include logs.source.systemd.conf
-@include logs.source.generic.conf
\ No newline at end of file
+@include logs.source.generic.conf
+@include logs.source.kube-audit.conf
```

Comment:
The order should be reverse for these two. The generic filter should be at the end .

Reply:
Fixed the ordering